cmd/kpod/load.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"github.com/containers/storage"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+const (
+	dockerArchive = "docker-archive:"
+	oci           = "oci:"
+)
+
+type loadOptions struct {
+	oci   bool
+	input string
+	quiet bool
+}
+
+var (
+	loadFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "oci",
```

Comment:
The user is handing the tool the archive, there is no choice.  
Think of it this way, this is like doing an editor

office resume.doc
office resume.ods

Would you make the user specify --doc, in order to read the .doc file or would the tool be smart enough to figure out that this is a .doc